static/js/server_events_dispatch.js

```diff
@@ -139,6 +139,12 @@ exports.dispatch_normal_event = function dispatch_normal_event(event) {
         settings_filters.populate_filters(page_params.realm_filters);
         break;
 
+    case 'custom_profile_fields':
+        page_params.custom_profile_fields = event.fields;
+        settings_profile_fields.populate_profile_fields(page_params.custom_profile_fields);
+        settings_profile_fields.report_success(event.opt);
```

Comment:
This should be called `op`, not `opt`, to match the style of other similar events.